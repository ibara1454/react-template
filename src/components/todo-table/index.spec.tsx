import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import TodoTable from '.';

describe('TodoTable', () => {
  it('should renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <TodoTable showType="all" />,
        </MemoryRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
