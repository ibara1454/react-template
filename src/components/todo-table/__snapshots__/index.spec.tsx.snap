// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoTable should renders correctly 1`] = `
Array [
  <div
    className="root"
  >
    <div>
      <header
        className="header"
      >
        <input
          className="newTodo"
          onKeyDown={[Function]}
          placeholder="What needs to be done?"
        />
        <label
          className="toggleAll toggled"
        >
          <input
            checked={true}
            onChange={[Function]}
            type="checkbox"
          />
        </label>
      </header>
      <section
        className="main"
      >
        <ul
          className="todoList"
        />
      </section>
    </div>
    <footer
      className="root"
    >
      <span
        className="count"
      >
        <strong>
          0 left
        </strong>
      </span>
      <ul
        className="filters"
      >
        <li>
          <a
            aria-current="page"
            className="link selected"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            All
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="link"
            href="/active"
            onClick={[Function]}
          >
            Active
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="link"
            href="/completed"
            onClick={[Function]}
          >
            Completed
          </a>
        </li>
      </ul>
    </footer>
  </div>,
  ",",
]
`;
